packages/flutter_tools/lib/src/migrate/migrate_compute.dart

```diff
@@ -0,0 +1,796 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
My suggestion is to refactor all of the code in this file into classes with well-defined interfaces that can then be easily unit-tested. The PR that does that should be landed before this one. Pausing my review here.